internal/ignition/ignition.go

```diff
@@ -39,7 +39,8 @@ import (
 	"github.com/openshift/assisted-service/pkg/auth"
 	"github.com/openshift/assisted-service/pkg/s3wrapper"
 	"github.com/openshift/assisted-service/pkg/staticnetworkconfig"
-	"github.com/pelletier/go-toml"
+
+	//"github.com/pelletier/go-toml"
```

Comment:
Can you remove the comment and added whitespace?

Reply:
fixed